FLEDGE.md

```diff
@@ -121,6 +121,7 @@ const myGroup = {
   'executionMode': ...,
   'trustedBiddingSignalsURL': ...,
   'trustedBiddingSignalsKeys': ['key1', 'key2'],
+  'trustedBiddingSignalsSlotSizeMode' : 'slot-size',
```

Comment:
Should this below, 

`'adSizes': {'size1': {width: width1, height: height1},`

reinforce that widths & heights are string valued? And probably also use realistic dimensions.

Reply:
I've updated the adSizes line to use strings, with values (And all 4 unit types we support).

Are you also suggesting we move 'trustedBiddingSignalsSlotSizeMode' down next to ad sizes?  Less sure about that - I think there's value in having all trustedBiddingSignals stuff grouped together, though not something I'd lose sleep over, one way or the other.

Reply:
Saw, TY, that and commented on it.

>Are you also suggesting we move 'trustedBiddingSignalsSlotSizeMode' down next to ad sizes? 

No, not sure how I gave that impression.